syne_tune/experiments/benchmark_definitions/fcnet.py

```diff
@@ -10,7 +10,7 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-from benchmarking.commons.benchmark_definitions.common import (
+from syne_tune.experiments.benchmark_definitions.common import (
```

Comment:
One of the main reasons of this refactoring is that most folks do not use this very useful methodology, because they somehow think this is only useful for benchmarking (i.e., comparing different HPO methods).

I kind of thought of a "benchmark" as an HPO problem to be addressed, but since this term seems to trigger the wrong association, I'll think of another one.

Reply:
Well no, I will keep this term, it is too ingrained. But I'll make it clear that a "benchmark" is nothing but a tuning problem.